### PR TITLE
Travis support

### PR DESCRIPTION
**Pull request by gauntface**
Part of prep for moving to web-push-libs.

### CODE REVIEW

**Comment by gauntface** [comment]
@x1ddos @wibblymat This PR removes drone and switches to Travis support - this should just make it a little easier for web-push-libs org to work with the library

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+go:
+  - 1.6.x
+  - 1.7.x
+  - 1.8.x
+  - master
+
+before_install:
+  - go get github.com/golang/lint/golint
+
+install:
```
Comment (x1ddos) [inline .travis.yml]: wouldn't `script` be more appropriate here?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - cd webpush
+
```
Comment (x1ddos) [inline .travis.yml]: remove empty line here and below?

**Review by x1ddos — CHANGES_REQUESTED** [review]
I would remove changes from webpush/encrypt.go in this PR to avoid mixing unrelated things.

**Review by x1ddos — APPROVED** [review]

**Pull request merged**